pkgs/applications/networking/gopher/sacc/default.nix

```diff
@@ -1,14 +1,15 @@
-{ lib, stdenv, fetchurl, ncurses
+{ lib, stdenv, fetchgit, ncurses
 , patches ? [] # allow users to easily override config.def.h
 }:
 
 stdenv.mkDerivation rec {
   pname = "sacc";
-  version = "1.02";
+  version = "1.03";
 
-  src = fetchurl {
-    url = "ftp://bitreich.org/releases/sacc/sacc-${version}.tgz";
-    sha512 = "18ja95cscgjaj1xqn70dj0482f76d0561bdcc47flqfsjh4mqckjqr65qv7awnw6rzm03i5cp45j1qx12y0y83skgsar4pplmy8q014";
+  src = fetchgit {
```

Comment:
`1.03` is also available via ftp and using a release tar ball is preferable over `fetchgit`.

Reply:
Okiedokie I'll revert